_test_unstructured_client/test__decorators.py

```diff
@@ -1,70 +1,98 @@
-import os
 import pytest
 
 from unstructured_client import UnstructuredClient
+from unstructured_client.models import shared
+from unstructured_client.models.errors import SDKError
 
 
-def get_api_key():
-    api_key = os.getenv("UNS_API_KEY")
-    if api_key is None:
-        raise ValueError("""UNS_API_KEY environment variable not set. 
-Set it in your current shell session with `export UNS_API_KEY=<api_key>`""")
-    return api_key
+FAKE_KEY = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 
 
 @pytest.mark.parametrize(
     ("server_url"),
     [
-        ("https://unstructured-000mock.api.unstructuredapp.io"), # correct url
+        ("https://unstructured-000mock.api.unstructuredapp.io"),  # correct url
         ("unstructured-000mock.api.unstructuredapp.io"),
         ("http://unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
         ("https://unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
         ("unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
-    ]
+    ],
 )
 def test_clean_server_url_on_paid_api_url(server_url: str):
     client = UnstructuredClient(
         server_url=server_url,
-        api_key_auth=get_api_key(),
+        api_key_auth=FAKE_KEY,
+    )
+    assert (
+        client.general.sdk_configuration.server_url
+        == "https://unstructured-000mock.api.unstructuredapp.io"
     )
-    assert client.general.sdk_configuration.server_url == "https://unstructured-000mock.api.unstructuredapp.io"
 
 
 @pytest.mark.parametrize(
     ("server_url"),
     [
-        ("http://localhost:8000"), # correct url
+        ("http://localhost:8000"),  # correct url
```

Comment:
On these kinds of comments, I'd be inclined to make this a "header" comment rather than a trailing one, and add a heading comment saying what the others are in contrast. Something like:
```python
# -- well-formed url --
"http://localhost:8000",
# -- url commonly ill-formed by users --
"localhost:8000",
...
```

Also note the parens are not necessary here since there is only the single parameter so those should come out. I'm surprised Black didn't do that automatically, maybe it's not running here.